### PR TITLE
implement rangeslider page

### PR DESCRIPTION
**Pull request by PhungNg**
## Description

Implement the rangeslider page

## Motivation and Context
Dividing the different components in Forms page into seperate pages.

## How Has This Been Tested?
Ran and updated existing test, and added new snapshot

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/App/ComponentsDocumentation/components/Rangeslider/index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        <ul className="list list-bullet">
+            <li>Use a text input instead when the user needs to select specific number.</li>
+        </ul>{/* Link to text-input */}
+
+        <h3>How to use sliders</h3>
+
+        {howToUse.map(({ type, component, text }, i) => (
+            <React.Fragment key={i}>
+                <div className={`slab slab-plain slab-border-${type} h-100`}>
+                    <h3 className="mt-0">Do{type === "error" && "n’t"}</h3>
```
Comment (VikingTristan) [inline src/App/ComponentsDocumentation/components/Rangeslider/index.js:23]: Cool operator :)

File: src/App/ComponentsDocumentation/components/Rangeslider/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+);
+
+const ContentGuidelines = () => (
+    <section>
+        <h2 id="content-guidelines">Content guidelines</h2>
+
+        <div className="slab slab-plain h-100">
+            <div className="d-flex flex-column align-items-center">
+                <div className="py-3 col-8">
+                    <div>{/* add label */}
```
Comment (VikingTristan) [inline src/App/ComponentsDocumentation/components/Rangeslider/index.js:43]: Add "TODO" if needed

File: src/App/ComponentsDocumentation/components/Rangeslider/index.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        <h2 id="javascript-method mb-3">Javascript method</h2>
+        <CodeTags type="secondary" code="dg.rangeslider.init()"/>
+        <p className="my-3"><a href="/get-started/for-developers" >Initalizes</a> rangeslider</p>
+    </section>
+);
+
+const Rangeslider = () => {
+    useEffect(() => {
+        window.dg.rangeslider.init();
+    });
```
Comment (VikingTristan) [inline src/App/ComponentsDocumentation/components/Rangeslider/index.js:78]: As you explained on slack :)

File: src/less/documentation-swedbankpay.less [hunk 4]
```diff
@@ -681,6 +681,20 @@
...
             }
 
+            &#no-tabs {
+                #undefined-ul {
+                    .active {
+                        a {
+                            display: none;
+                        }
+                    }
+                }
```
Comment (VikingTristan) [inline src/less/documentation-swedbankpay.less:691]: Interesting hack :)

**Comment by VikingTristan** [comment]
Re init on state change

**Comment by PhungNg** [comment]
Created a new PR

**Pull request closed without merging**